trusted-firmware-m/secure_fw/partitions/crypto/tfm_crypto_secure_api.c

```diff
@@ -1703,7 +1703,7 @@ psa_status_t psa_key_derivation_key_agreement(
 psa_status_t psa_generate_random(uint8_t *output,
                                  size_t output_size)
 {
-#ifdef TFM_CRYPTO_GENERATOR_MODULE_DISABLED
+#ifdef TFM_CRYPTO_RNG_MODULE_DISABLED
```

Comment:
just a note @hakonfam @oyvindronningstad , this is different upstream, now, so we will need to attend this on next upmerge.